layouts/_default/baseof.html

```diff
@@ -9,7 +9,7 @@
 
   {{- partial "global_navbar.html" . -}}
   {{ $path := split $.Source.File.Path .Site.Params.PathSeparator }}
-  {{ if or (eq (index $path 0) "events") (eq .Type "events") }}
```

Comment:
I think it would be better to actually test this against the right types; the type of an event is "event" not "events", which is why this is wrong.

I don't think this works the way you think it does (I haven't been able to test it locally) but shouldn't it a test for the path of "events" but not if type is "events", because the Events page is in that path but has the type of Events.